shared_infra/dynamo_write_heartbeat/src/dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,26 @@
+# -*- encoding: utf-8 -*-
+"""
+Run a no-op write on DynamoDB (delete a non existent item) to be run in a scheduled lambda to
+make DynamoDB scale down when there would otherwise be zero throughput and it otherwise wouldn't.
+
+TABLE_NAMES: comma separated list of dynamoDb tables to call
+"""
+
+import boto3
+import os
+
+from wellcome_aws_utils.lambda_utils import log_on_error
+
+
+@log_on_error
+def main(event, context, dynamodb_client=None):
+    try:
+        table_names = [t.strip(' ') for t in os.environ['TABLE_NAMES'].split(',')]
```

Comment:
minor: calling `t.strip()` without any arguments should already strip whitespace for you